.github/workflows/ci.yml

```diff
@@ -22,7 +22,8 @@ jobs:
         ruby_version:
         - 2.5
         - 2.7
-        - jruby-9.2.11.1
+        - 3.0
```

Comment:
It is not directly related to this PR, but just wondering why we are missing Ruby 2.6 on CI matrix? :thinking:

Reply:
I wondered that too. It does seem worth adding. I could put it in this PR if you like.

Reply:
Ruby 2.6 is not included in the CI matrix because we, the maintainers, felt that it is unnecessary to test all supported Ruby versions since there are hardly any backwards incompatible changes between the recent versions of Ruby 2.x (v2.4 - v2.7).
We therefore test only the lowest Ruby version under active maintenance and the current Ruby version.

*However, I personally am not going to ask you to remove Ruby 3.0 because it is a major version bump*.